cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/internal/BootstrapCoordinator.scala

```diff
@@ -135,6 +135,7 @@ private[akka] final class BootstrapCoordinator(discovery: SimpleServiceDiscovery
 
     case SimpleServiceDiscovery.Resolved(name, contactPoints) ⇒
       serviceName = name
+      log.info("Located service members with name: [{}].", serviceName)
```

Comment:
```
log.info("Located service members with name: [{}]: [{}]", serviceName, contactPoints.mkString(", "))
```

?

Later, I think we should revisit the log level of some of the logging. It's rather verbose on info level now and might be better to turn some of it into debug.